eox_audit_model/tests/tests_tasks.py

```diff
@@ -0,0 +1,89 @@
+"""This file contains all the test for the tasks file.
+
+Classes:
+    TestCreateAuditRegister: Test AuditModel.
+"""
+import logging
+import traceback
+
+from django.test import TestCase
+
+from eox_audit_model.constants import Status
+from eox_audit_model.models import AuditModel
+from eox_audit_model.tasks import create_audit_register
+
+LOG = logging.getLogger(__name__)
+
+
+class TestAuditModel(TestCase):
+    """Test cases for the model AuditModel."""
+
+    def setUp(self):
+        """Setup common conditions for every test case"""
+        def valid_method(a, b, c, d):
+            """Execute a mathematic operation."""
+            LOG.info('This is an info message')
+            return (a + b + c) / d
+
+        self.valid_method = valid_method
+
+    def test_valid_method(self):
```

Comment:
can you add a test for a method that raises an exception?

Reply:
I have to change the name, the task doesn't execute any method

Reply:
and there is already a test for the exception case,  https://github.com/eduNEXT/eox-audit-model/blob/79e8668ee1bd524fa087e9061c5587763aa66f54/eox_audit_model/tests/tests_models.py#L223